exercises/binary-search-tree/binary_search_tree.py

```diff
@@ -0,0 +1,8 @@
+class TreeNode(object):
+    def __init__(self, value):
+        pass
+
+
+class BinarySearchTree(object):
+    def __init__(self):
+        pass
```

Comment:
Can you also include skeletons for the `add` and `search` methods here?

Reply:
Done